src/graphql/v1/common.js

```diff
@@ -120,12 +137,21 @@ module.exports.findResourcesByReference = async (parent, args, context, info, re
  * @param {string} resourceType
  * @return {Promise<Resource[]>}
  */
-// eslint-disable-next-line no-unused-vars
 module.exports.getResources = async (parent, args, context, info, resourceType) => {
+    /**
+     * @type {SimpleContainer}
+     */
+    const container = context.container;
+    assertTypeEquals(container, SimpleContainer);
```

Comment:
This check happens in other functions as well.  Should we convert this to a class/module and do a single check?

Reply:
the graphql stuff is not properly IoC yet since it should not get the container but the individual services.  But we have to get rid of graphv1 first.